recipe/meta.yaml

```diff
@@ -36,13 +38,25 @@ requirements:
 test:
```

Comment:
Would this be an appropriate place to use `{{ pin_compatible('openssl') }}` and should it also bear the `# [not win]` flag?

Reply:
Here we need `- openssl # [not win]`
Openssl has a run_exports https://github.com/AnacondaRecipes/openssl-feedstock/blob/master/recipe/meta.yaml#L24 which makes that when openssl is used in host, the equivalent of `pin_compatible('openssl', max_pin='x')` is implicitly added to run. By choice, we usually still refer the host package in run when that is the case. Here especially, that was done to ensure openssl 3 was used. However for windows, since openssl is not in host, this adds an unbounded dependency on openssl.